pom.xml

```diff
@@ -63,7 +63,7 @@
     <version.com.github.nmorel.gwtjackson>0.14.2</version.com.github.nmorel.gwtjackson>
     <version.com.github.nmorel.gwtjackson.restprocessor>0.4.1</version.com.github.nmorel.gwtjackson.restprocessor>
     <version.org.gwtbootstrap3>0.9.4</version.org.gwtbootstrap3>
-    <version.org.wildfly.wildfly-dist>10.1.0.Final</version.org.wildfly.wildfly-dist>
+    <version.org.wildfly.wildfly-dist>10.0.0.Final</version.org.wildfly.wildfly-dist>
```

Comment:
we probably still want to push the latest WildfFly final version, despite also being compatible with 10.0